pkg/util/log/go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/DataDog/datadog-agent/pkg/util/log
+
+go 1.14
+
+require (
+	github.com/cihub/seelog v0.0.0-20170130134532-f561c5e57575
```

Comment:
If we're doing an override for this module in the main agent's `go.mod`, do we need the same versioning here? I didn't go through the whole list but it also would be worth it to check that our main `go.mod` does not contain any packages that are no longer need to be used/referred directly (e.g. `github.com/cihub/seelog`)

Reply:
> If we're doing an override for this module in the main agent's go.mod, do we need the same versioning here?

We do need it when this package is consumed externally, since you don't 'see' the main `go.mod` file then.

> I didn't go through the whole list but it also would be worth it to check that our main go.mod does not contain any packages that are no longer need to be used/referred directly (e.g. github.com/cihub/seelog)

We have a `go mod tidy` check that does that on Gitlab, seelog is used in other places unfortunately (see `git grep "github.com/cihub/seelog" {pkg,cmd}/**.go`).